apps/andi/assets/css/app.scss

```diff
@@ -76,6 +76,14 @@ body {
   margin-right: 0;
 }
 
+.btn--test {
+  @extend .btn--cancel;
```

Comment:
This `extend` violates the pattern of BEM, though I've noticed it's been used in other places. I think the inheritance going on here will just become increasingly confusing vs just having blocks and modifiers. Your call on whether to clean this up or not.